.travis.yml

```diff
@@ -20,6 +20,6 @@ before_script:
 
 script:
 # Test that notebooks run successfully
-- find "$PWD" -name "*.ipynb" -exec sh test/test_notebook.sh "{}" \;
+- find "$PWD" -name "*.ipynb" -exec sh test/test_notebook.sh "{}" +
```

Comment:
Should this be `\+`

Reply:
No, the `\` is only required before the `;` in the previous state so that it doesn't get picked up by the shell